samples/image-uploader/src/components/ProgressView/index.js

```diff
@@ -1,25 +1,30 @@
-import React from "react"
-import "./progress-view.css"
+import React from 'react';
+import tokens from '@contentful/forma-36-tokens';
+import { Icon } from '@contentful/forma-36-react-components';
+import './progress-view.css';
 
 export default function UploadView(props) {
-  const imageUrl = props.imageUrl || `${props.base64Prefix},${props.base64Data}`
+  const imageUrl = props.imageUrl || `${props.base64Prefix},${props.base64Data}`;
+  const isSVG = imageUrl.endsWith('.svg') || imageUrl.includes('svg+xml');
 
-  const background = {
-    backgroundImage: `url(${imageUrl})`
-  }
+  const progressViewStyles = {
+    backgroundColor: tokens.colorElementLight
+  };
 
   // Pass upload progress as CSS variable so we can adjust the size of progress components
   const uploadProgress = {
-    "--uploadProgress": `${props.uploadProgress}%`
-  }
+    '--uploadProgress': `${props.uploadProgress}%`
+  };
 
   return (
     <div className="viewport">
-      <main className="progress-view" style={background}>
+      <main className="progress-view" style={progressViewStyles}>
+        {!isSVG && <img className="uploaded-image" src={imageUrl} alt="image being uploaded" />}
```

Comment:
Why not just make this a ternary instead of having two different conditions?

Reply:
voila